inception-project/inception
# #3215 - Reorganize CAS storage configuration properties

**What's in the PR**
- Renamed CasStorageProperties to CasStorageCacheProperties
- Introduce new CasStorageProperties where the compression for CASes can be configured
- Update documentation

**How to test manually**
* Set the properties as documented and see if they take effect (basically using a debugger)

**Automatic testing**

**Documentation**
